gcloud/search/document.py

```diff
@@ -0,0 +1,303 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Define API Document."""
+
+import datetime
+
+import six
+
+from gcloud._helpers import UTC, _RFC3339_MICROS
+from gcloud.exceptions import NotFound
+
+
+class StringValue(object):
```

Comment:
These could just be named tuples right? (Can also add default values, e.g. `value_type` here, to a `namedtuple`: http://stackoverflow.com/a/18348004/1068170)